REST_API_DOCS.md

```
@@ -0,0 +1,1802 @@
+# Cloud Community REST API Documentation
+
+본 문서는 **Cloud Community** 프로젝트의 모든 API 명세를 담고 있다
+
+### **내가 좋아요한 게시글 목록 조회**
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)